### PR TITLE
fix the linux/arm/v7 build

### PR DESCRIPTION
**Pull request by nextgens**
## What type of PR?

bug-fix

## What does this PR do?

The arm builder is running aarch64 ... and there is no package for arm/v7

### CODE REVIEW

**Comment by nextgens** [comment]
I have tested it locally on the builder and it seems to do the job

File: core/base/Dockerfile [hunk 1]
```diff
@@ -8,11 +8,13 @@ ENV TZ=Etc/UTC LANG=C.UTF-8
...
+ARG TARGETPLATFORM
 
 RUN set -euxo pipefail \
   ; addgroup -Sg ${MAILU_GID} mailu \
   ; adduser -Sg ${MAILU_UID} -G mailu -h /app -g "mailu app" -s /bin/bash mailu \
   ; apk add --no-cache bash ca-certificates curl python3 tzdata \
+  ; if [[ "$TARGETPLATFORM" == 'linux/arm/v7' ]]; then exit;fi \
   ; machine="$(uname -m)" \
   ; if [[ "${machine}" == x86_64 || "${machine}" == armv8* || "${machine}" == aarch64 ]] \
   ; then \
```
Comment (ghostwheel42) [inline core/base/Dockerfile:20]: I think this early exit will confuse people adding more commands at the end.
Why not just do it like this:
```
  ; [[ "$TARGETPLATFORM" != 'linux/arm/v7' ]] \
    && machine="$(uname -m)" \
    && [[ "${machine}" == x86_64 || "${machine}" == armv8* || "${machine}" == aarch64 ]] \
    && apk add --no-cache --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing hardened-malloc
```

**Review by ghostwheel42 — CHANGES_REQUESTED** [review]

File: core/base/Dockerfile (hunk 1 above)
Comment (nextgens) [reply core/base/Dockerfile:20]: Because the shell isn't bash, it's sh

Comment (nextgens) [reply core/base/Dockerfile:20]: But even with the SH syntax, the ``set -euxo pipefail`` will produce an error as test returns false

Comment (ghostwheel42) [reply core/base/Dockerfile:20]: I don't see bash specifics in my proposal. But yes, it needs to be negated:
```
  ; machine="$(uname -m)"
    ! [[ "${TARGETPLATFORM}" != linux/arm/v7 && ( "${machine}" == x86_64 || "${machine}" == armv8* || "${machine}" == aarch64 ) ]] \
    || apk add --no-cache --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing hardened-malloc
```

If you don't like this please wrap the TARGETPLATFORM if/then/fi around the other lines, and don't exit early.

**Comment by nextgens** [comment]
@ghostwheel42 ready for another review

**Review by ghostwheel42 — DISMISSED** [review]
lgtm

**Comment by Diman0** [comment]
bors try

**Review by Diman0 — APPROVED** [review]
lgtm

**Pull request merged**